navigator/backend/src/test/scala/com/digitalasset/navigator/backend/query/ContractFilterSpec.scala

```diff
@@ -99,6 +102,10 @@ class ContractFilterSpec extends FlatSpec with Matchers {
 
   testOr(List("argument.foo" -> "bar", "argument.int" -> "1"), contracts)
 
+//  testAnd(List("agreementText" -> ""), List(contract2, contract3))
```

Comment:
do we still need this line?

Reply:
Thanks for the catch. I fixed the code to make this test work and enabled it.